pom.xml

```diff
@@ -24,7 +24,7 @@
     <project_organization>fcrepo</project_organization>
     <fcrepo-storage-ocfl.version>6.4.0-SNAPSHOT</fcrepo-storage-ocfl.version>
     <!-- Dependency version properties -->
-    <activemq.version>5.16.3</activemq.version>
+    <activemq.version>5.16.7</activemq.version>
```

Comment:
Wondering if we should try for a more recent version like `5.17.6` or `5.18.3`?

Reply:
If we update, I think it would make sense to go to 5.18.3 directly.

Reply:
My only worry about that was the possible breaking changes by jumping 2 major versions rather than just minor versions.

Reply:
I would assume that a breaking change would have bumped the major version from 5 to 6, but we will have to test it out anyways. There is no rush on this now.

Reply:
According to https://docs.aws.amazon.com/amazon-mq/latest/developer-guide/activemq-version-management.html 5.16 is a major version, as it 5.18. They use defferent terminology to the sem,ver spec awkwardly. 

I'm happy to trial 5.18 but can we push this PR as is now and do that in a future release along with a bunch of other updates from Dependapot which have been missed. I'll open another ticket about that

Reply:
Fine for me